README.md

```diff
@@ -1,2 +1,37 @@
 # programapi
-Program API
+
+This project downloads, processes, saves, and serves the static JSON files containing details of accepted speakers and submissions via an API.
```

Comment:
Do you know where we will use this data later? Is it for the website?

Reply:
Mainly for the website. Also, maybe we can use this in the Discord bot as well

Reply:
Added some more explanation